app/App/Account/index.js

```diff
@@ -185,32 +185,6 @@ class Main extends React.Component {
     }, 3000)
   }
 
-  accountSort (accounts, a, b) {
```

Comment:
sorry just realized, why does this function take a dictionary and two ids? can't it just take two accounts and sort a list like the usual sort function? something like:

```
import { accountSort as byTimestamp } from 'resources/domain/account'

...

const accounts = [ ... ]
const sorted = accounts.sort(byTimestamp)
```